airbyte-integrations/connectors/destination-gcs/src/test-integration/java/io/airbyte/integrations/destination/gcs/GcsCsvDestinationAcceptanceTest.java

```diff
@@ -38,7 +38,9 @@ public GcsCsvDestinationAcceptanceTest() {
   protected JsonNode getFormatConfig() {
     return Jsons.jsonNode(Map.of(
         "format_type", outputFormat,
-        "flattening", Flattening.ROOT_LEVEL.getValue()));
+        "flattening", Flattening.ROOT_LEVEL.getValue(),
+        // test the csv format without the compression
```

Comment:
should we have a test for compressed output? Or is the idea that we test csv+uncompressed, and jsonl+compressed, and there's no benefit to testing the full cross-product?

Reply:
I'd prob say all combos would be preferable in general, is there a chance that either of those missing combos could have an uncaught bug from future changes? (I dont know this code nearly as well as you, I'm actually curious) if there isnt, then this seems fine to me though

Reply:
Good point.

I was trying to avoid adding a full length integration test just to test the compression to keep the total integration test run short. So I did the following test combination:
- S3 CSV - with compression
- S2 JSONL - no compression
- GCS CSV - no compression
- GCS JSONL - with compression

But I just realized that the tests are fast in general now (8 min and 13 min). So adding more tests won't severely slow them down.

I will think about how to easily test both formats with and without compressions for each connector independently.